Failed to execute command "mvn clean test"

### Problem to solve
Test execution in terminal with `$ mvn clean test` command causes following error:
```
[INFO] Running sitemap.tests.SitemapTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.169 s <<< FAILURE! - in sitemap.tests.SitemapTest
[ERROR] testAvailabilityOfAllSitemapLinks(sitemap.tests.SitemapTest)  Time elapsed: 0.166 s  <<< ERROR!
java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
        at sitemap.tests.SitemapTest.checkSitemapIsAvailable(SitemapTest.java:26)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
        at sitemap.tests.SitemapTest.checkSitemapIsAvailable(SitemapTest.java:26)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   SitemapTest.checkSitemapIsAvailable:26 » NoClassDefFound javax/xml/bind/JAXBEx...
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```
### Possible solution
Find and install JAXB libs, something like that https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j

### Current configuration on mac is
```
$ java -version
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
$ mvn -version
mvn -version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /usr/local/Cellar/maven/3.6.3_1/libexec
Java version: 13.0.2, vendor: N/A, runtime: /usr/local/Cellar/openjdk/13.0.2+8_2/libexec/openjdk.jdk/Contents/Home
Default locale: ru_RU, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```
### Solution
This guy's solution works for my config https://stackoverflow.com/a/56509710/8091236
```
        <!-- replace dependencies that have been removed from JRE's starting with Java v11 -->
        <dependency>
            <groupId>javax.xml.bind</groupId>
            <artifactId>jaxb-api</artifactId>
            <version>2.2.8</version>
        </dependency>
        <dependency>
            <groupId>com.sun.xml.bind</groupId>
            <artifactId>jaxb-core</artifactId>
            <version>2.2.8-b01</version>
        </dependency>
        <dependency>
            <groupId>com.sun.xml.bind</groupId>
            <artifactId>jaxb-impl</artifactId>
            <version>2.2.8-b01</version>
        </dependency>
        <!-- end replace dependencies that have been removed from JRE's starting with Java v11 -->
```

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>java-sitemap-checker</groupId>
     <artifactId>java-sitemap-checker</artifactId>
-    <version>1.2-SNAPSHOT</version>
+    <version>1.3-SNAPSHOT</version>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
@@ -70,7 +70,23 @@
             <scope>test</scope>
         </dependency>
 
-
+        <!-- replace dependencies that have been removed from JRE's starting with Java v11 -->
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.8</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-core</artifactId>
+            <version>2.2.8-b01</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+            <version>2.2.8-b01</version>
+        </dependency>
+        <!-- end replace dependencies that have been removed from JRE's starting with Java v11 -->
     </dependencies>
 
 </project>

